client/app/components/routes/operations/Operations.tsx

```diff
@@ -1,6 +1,6 @@
 import { GridRowsProp } from "@mui/x-data-grid";
-import DataGrid from "@/components/datagrid/DataGrid";
-import { fetchAPI } from "@/utils/api";
+import DataGrid from "../../datagrid/DataGrid";
```

Comment:
@shon-button, my app couldn't find these imports until I changed them to the .. vs @. Have I got something set up wrong?

Reply:
I have no issues on this branch when using import for any of the tsconfig.json paths such as: import DataGrid from "@/components/datagrid/DataGrid";  Perhaps, it's a cache issue on your side needing a re-start of the dev server?

Reply:
For client/app/components/routes/operations/Operations.tsx imports:

**yarn dev** works with either:  "@/components/datagrid/DataGrid";  "@/app/components/datagrid/DataGrid"; "../../datagrid/DataGrid";
**yarn test** works with only: "@/app/components/datagrid/DataGrid";  "../../datagrid/DataGrid";

**yarn test** with import DataGrid from "@/components/datagrid/DataGrid"; errors with: Cannot find module '../../../../components/datagrid/DataGrid' from 'app/components/routes/operations/Operations.tsx'

Reply:
Suggest using only path alias @/* for imports to ensure proper jest test functionality.

e.g.
❌  "@/components/datagrid/DataGrid";
✔️  "@/app/components/datagrid/DataGrid";


❌  "@/utils/api";
✔️  "@/app/utils/api";

Reply:
Of interest?
[Next.js with React Testing Library, Jest, TypeScript](https://www.youtube.com/watch?v=AS79oJ3Fcf0)
[repo](https://github.com/gitdagray/next-testing-intro/tree/main)

Reply:
If agreement to use one path alias @/* for imports then the following files can be updated to reflect the single path alias:
- tsconfig.json\paths
- jest.config.mjs\ moduleNameMapper